### PR TITLE
fix errors in Makefile

### PR DESCRIPTION
**Pull request by b00f**
* remove gogoproto from tools (NOTE; gogoproto is not an executable. look here: https://github.com/gogo/protobuf/blob/master/gogoproto/Makefile)
* fix make `get_check` and `get_vendor_deps` errors

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -15,7 +15,7 @@
   name = "github.com/btcsuite/btcd"
```
Comment (melekes) [inline Gopkg.lock]: can you exclude the updates from this PR?

**Comment by melekes** [comment]
@b00f thank you for checking the Makefile 👍

File: Gopkg.lock (hunk 1 above)
Comment (b00f) [reply Gopkg.lock]: what updates?

Comment (melekes) [reply Gopkg.lock]: fix_errors_in_makefile_by_b00f_ _pull_request__2146_ _tendermint_tendermint_

Comment (b00f) [reply Gopkg.lock]: I had error on running `dep ensure`. Removing `Gopkg.lock` and running the `dep ensure` fixed the errors.

**Comment by ebuchman** [comment]
Fixed in https://github.com/tendermint/tendermint/pull/2152/commits/c5c2b9601f02747d50e70f54a71f40270a659956, thanks

**Comment by b00f** [comment]
@ebuchman I still have same errors. Could you please check this command: `make check`
There is no `gogoproto` as executable in PATH.
Please check here: https://github.com/gogo/protobuf/blob/master/gogoproto/Makefile

**Comment by melekes** [comment]
@b00f sorry, fixed in https://github.com/tendermint/tendermint/pull/2198

**Pull request closed without merging**